src/Components/CreateImageWizard/validators.ts

```diff
@@ -67,6 +67,25 @@ export const isFileSystemConfigValid = (partitions: Partition[]) => {
   return duplicates.length === 0;
 };
 
+export const isUserNameValid = (userName: string) => {
+  if (userName === undefined) return false;
+  const isLengthValid = userName.length >= 1 && userName.length <= 32;
+  const isNotDotOrDotDot = userName !== '.' && userName !== '..';
```

Comment:
Is this not included in the regex isPatternValid? Deleting this condition did not allow adding the dots and double dots into the username. :thinking:

Reply:
yes, thank you for that!